pkg/watch/mux.go

```diff
@@ -39,6 +39,8 @@ const incomingQueueLength = 25
 // Broadcaster distributes event notifications among any number of watchers. Every event
 // is delivered to every watcher.
 type Broadcaster struct {
+	// TODO: see if this lock is needed now that new watchers go through
+	// the incoming channel.
```

Comment:
Basically I'm being lazzzzy here because I need to stop being lazzzzy about my PR reviews. :-(